### ISSUE TITLE
Add a config option to prevent combatant cards pop-out behaviour during hover-over

### ISSUE THREAD

**Issue by thazzunk** [death-save/combat-carousel#151]
When you hover over a combatant card in the carousel, the card will pop-out, making the image appear larger and moving it down, making room to show the name above the creature. In addition, you can then hover over the name to access the controls for 'toggle visibility', 'mark defeated', and 'remove from encounter'. 

This pop-out interface is visually impressive, as it results in a clean-looking UI when not interacting with the carousel. However, I find this behaviour a bit jarring and functionally impractical. I would prefer it if there was an option in the configuration to change the behaviour to the following:
1. the card image does not change size on hover-over
2. the card image does not get displaced downward on hover-over
3. I could see the names at all times
4. I could see the 'toggle visibility', 'mark defeated', and 'remove from encounter' controls at all times

Alternatives for 3 and 4 would be to have the names or controls only appear on hover-over, but they would appear above and/or below the card image, without displacing it. This would help maintain the clean-looking UI. I don't think it would be jarring, so long as the image stays fixed as described by 1 and 2. 

In summary, I think a more 'static' UI is more practical and would like to have an option which allows me to use one.
